signing-providers/src/hw.js

```diff
@@ -99,15 +102,10 @@ export class HW {
         await this.provider.init();
 
         const message = new SignableMessage({
-            message: Buffer.from("hello")
+            message: Buffer.from("erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6thexample-auth-token{}")
```

Comment:
is this change expected?

Reply:
Not expected, thanks for the catch :+1: :100: